repos/system_upgrade/el8toel9/actors/checkvdo/actor.py

```diff
@@ -0,0 +1,25 @@
+from leapp.actors import Actor
+from leapp.libraries.actor.checkvdo import check_vdo
+from leapp.models import Report, InstalledRedHatSignedRPM
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class CheckVdo(Actor):
+    """
+    Check if vdo devices need to be migrated to lvm management.
+    """
```

Comment:
Usually what is in the README.md is expected to be here - no problem that it's long :) just not sure the README.md file should be installed on the system to be honest. I know that ansible is installing README.md files under site-packages but not sure it's good practice. so the change in the specfile should be probably added as well. but in case you move the info just here, the problem is solved. you can keep MD format (just remove `<u></u>` tags)

Reply:
@pirat89 
Commit de4b5e5 moves the README.md contents to actor.py.